zulip_bots/zulip_bots/lib.py

```diff
@@ -166,7 +166,7 @@ def send_reply(self, message: Dict[str, Any], response: str, widget_content: Opt
         if message['type'] == 'private':
             return self.send_message(dict(
                 type='private',
-                to=[x['email'] for x in message['display_recipient'] if self.email != x['email']],
+                to=[x['email'] for x in message['display_recipient']],
```

Comment:
Can we change this to use user IDs instead while we're updating it?

Reply:
Right!